frontend/src/store/newUserComparatorState.ts

```diff
@@ -0,0 +1,180 @@
+import { ref } from 'vue';
```

Comment:
Hi Jota, we need to rename this file.  If you check other store files, it needs to start with upper case letter. And this is actually not new user, it's the new user access. Maybe it's better to call it `GrantUserAccessState.ts`. And you might want to rename your variables in this file, instead of calling them new users, change to new user access.

Reply:
sure, I'll make the change.